### DISCUSSION TITLE
Setup

### DISCUSSION THREAD

**Discussion by dwlr09** [Security-Onion-Solutions/securityonion#4877] [Unsupported Versions]
Is there a way to re-run setup thorough CLI?

**Comment by Git-Me-Some-Hub** [comment]
It is in the documentation somewhere, but this is the command. 
sudo SecurityOnion/setup/so-setup iso

**Reply by Git-Me-Some-Hub** [reply]
https://docs.securityonion.net/en/2.3/installation.html
